examples/julia_logo.jl

```diff
@@ -0,0 +1,159 @@
+using Javis
+using DelimitedFiles
+using OffsetArrays, FFTViews # (not strictly necessary, but make DFT indexing more intuitive)
+
+# Import the discrete fourier transform of our points and recast them in their original complex form
+ps = readdlm("julia_logo.csv", ',')
+cs = complex.(ps[:, 1], ps[:, 2]) |> FFTView
+
+vid = Video(800, 500)
+
+function ground(args...)
+    background("black")
+    sethue("black")
+    circle(Point(0, 0), 200, :fill)
+end
+
+function object(
+    p = O,
+    color = "white";
+    size = 25,
+    edge = "solid",
+    style = :stroke,
+    opacity = 1 / 3,
+)
+    sethue(color)
+    setdash(edge)
+    setopacity(opacity)
+    circle(p, size, style)
+    return p
+end
+
+function draw_line(
+    p1 = O,
+    p2 = O;
+    color = "white",
+    action = :stroke,
+    edge = "solid",
+    linewidth = 3,
+)
+    sethue(color)
+    setdash(edge)
+    setline(linewidth)
+    line(p1, p2, action)
+end
+
+function draw_path!(path, pos, color)
+    sethue(color)
+    push!(path, pos)
+    draw_line.(path[2:end], path[1:(end - 1)]; color = color)
+end
+
+import Javis.Point
+Point(c::Complex) = Point(real(c), imag(c))
+distance(p1::Point, p2::Point) = sqrt((p2.x - p1.x)^2 + (p2.y - p1.y)^2)
+
+# Draw a vector between points p1 and p2
+function vector(p1, p2; color = "white", opacity = 1 / 2, linewidth = 1, tipsize = 0.1)
```

Comment:
Cool, didn't even know it existed. I actually spent an excessive amount of time getting the arrow tip right so I'm more than happy to replace it

Reply:
Not really. It gives an error for trying to connect two identical points. I'm not sure if it is due to an initialization problem or if it's because at any point during the animation two points actually coincide (which might actually happen given how the fourier series are constructed)